src/tasks/set-connection-variables/set-connection-variables-v2/task.json

```diff
@@ -0,0 +1,83 @@
+{
+  "id": "d91997db-d6a0-42ed-ad3a-954b80e043ef",
+  "name": "PowerPlatformSetConnectionVariables",
+  "friendlyName": "Power Platform Set Connection Variables",
+  "description": "Sets BuildTools.* variables to provide custom script tasks access to use the Service Connection as a single source of truth.",
+  "author": "Microsoft",
+  "helpMarkDown": "[More Info](https://aka.ms/buildtoolsdoc)",
+  "category": "Utility",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [
+    "PowerPlatformTool"
+  ],
+  "version": {
+    "Major": "0",
+    "Minor": "1",
+    "Patch": "0"
+  },
+  "preview": false,
+  "minimumAgentVersion": "1.95.0",
+  "instanceNameFormat": "Power Platform Set Connection Variables $(message)",
+  "inputs": [
```

Comment:
not blocking for this PR, but: the task does emit output variables in its index.ts, shouldn't those also be listed in this task.json?
https://github.com/Microsoft/azure-pipelines-task-lib/blob/master/tasks.schema.json